flang/docs/index.md

```diff
@@ -16,6 +16,22 @@ on how to get in touch with us and to learn more about the current status.
    ReleaseNotes
 ```
 
+# Using Flang
+
+```{eval-rst}
+.. toctree::
+   :titlesonly:
+
+   GettingStarted
+   FlangCommandLineReference
+   FortranStandardsSupport
+   Extensions
+   Directives
+   OpenMPSupport
+   Real16MathSupport
+   OptionComparison
```

Comment:
The word `Flang` in `OptionComparison` refers Classic Flang. It should be remain in the "Design Documents" section.
(And it is better to reword it to avoid confusion)

Reply:
I submitted a renaming PR: https://github.com/llvm/llvm-project/pull/171558